### DISCUSSION TITLE
想知道你对UmiJS的未来的看法？关于React的最新NextJS发展。

### DISCUSSION THREAD

**Discussion by xXAvoraXx** [umijs/umi#10871] [General]
你好，我喜欢使用UmiJS，它有一个干净的层次结构和整齐的代码流。我想知道你对create-next-app有什么看法，它最近一直让前端行业很忙。UmiJS在我所在的地区并不流行，也不出名，我觉得很难描述UmiJS的优势，而业界普遍倾向于NextJS。

**Comment by BoyYangzai** [comment]
https://umijs.org/docs/introduce/introduce#nextjs
参考官网
自己的思考：
我个人认为UmiJs在企业内部的应用优势更大，它可以根据我们的需求做出相应的调整与改变，例如蚂蚁的Bigfish就是在Um的基础上打通了蚂蚁内部应用，如雨燕等(我没记错的话，之前在蚂蚁实习的时候有看bigfish的文档)
现在的preset-umi和umi/max更像是一个高达，你可以在此基础上为自己的公司和团队搭建属于自己的更强力的装备,当然绝大部分需求Umi团队已经完成/在进行了
